data/electronToChromium.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  "1.5": 54,
```

Comment:
Since 1.5 is not out yet, and depending on how long it takes to upgrade Electron to Chrome 54, Electron may skip it 54 altogether and go right to Chrome 55 which is due to hit stable very soon.